src/rest-api/endpoints/management-endpoint/migration-endpoint.md

```diff
@@ -0,0 +1,215 @@
+```
+
+```json
+{
+  "entity-type": "migration",
+  "id": "trash-storage",
+  "description": "Migration of in the trash storage model",
+  "descriptionLabel": "migration.trash-storage",
+  "status": {
+    "state": "property",
+    "step": null,
+    "startTime": 0,
+    "pingTime": 0,
+    "progressMessage": null,
+    "progressNum": 0,
+    "progressTotal": 0,
+    "running": false
+  },
+  "steps": []
+}
+```
+
+## Probe a Migration Object
+
+```
+POST /management/migration/MIGRATION_ID/probe
+```
+
+### Path Parameters
+
+| Parameter Name   | Type       | Description      |
+| ---------------- | ---------- | ---------------- |
+| **MIGRATION_ID** | **string** | The migration id.|
+
+### Response
+
+If successful, probes the current state of a migration by analyzing persistent data, and sets it as the new current state, then returns a JSON representation of the requested migration object.
+
+### Status Codes
+
+- 200 _OK_ - Success.
+- 404 _Not Found_ - Migration with the given `MIGRATION_ID` does not exist.
+
+### Sample
+
+```curl
+curl -X POST -u Administrator:Administrator \
+http://localhost:8080/nuxeo/api/v1/management/migration/trash-storage/probe
+```
+
+```json
+{
+  "entity-type": "migration",
+  "id": "trash-storage",
+  "description": "Migration of in the trash storage model",
+  "descriptionLabel": "migration.trash-storage",
+  "status": {
+    "state": "property",
+    "step": null,
+    "startTime": 0,
+    "pingTime": 0,
+    "progressMessage": null,
+    "progressNum": 0,
+    "progressTotal": 0,
+    "running": false
+  },
+  "steps": []
+}
+```
+
+## Run a Migration
+
+```
+POST /management/migration/MIGRATION_ID/run
+```
+
+### Path Parameters
+
+| Parameter Name   | Type       | Description      |
+| ---------------- | ---------- | ---------------- |
+| **MIGRATION_ID** | **string** | The migration id.|
+
+### Response
+
+If successful, probes the current state of a migration by analyzing persistent data, sets it as the new current state, then runs the migration asynchronously.
+
+### Status Codes
+
+- 202 _Accepted_ - Success.
+- 400 _Bad Request_ - Unknown migration `MIGRATION_ID`.
+
+### Sample
+
+```curl
+curl -X POST -u Administrator:Administrator \
+http://localhost:8080/nuxeo/api/v1/management/migration/trash-storage/run
+```
+
+## Run a Migration Step
+
+```
+POST /management/migration/MIGRATION_ID/run/MIGRATION_STEP_ID
+```
+
+### Path Parameters
+
+| Parameter Name        | Type       | Description           |
+| --------------------- | ---------- | --------------------- |
+| **MIGRATION_ID**      | **string** | The migration id.     |
+| **MIGRATION_STEP_ID** | **string** | The migration step id.|
+
+### Response
+
+If successful, runs the migration step asynchronously.
+
+### Status Codes
+
+- 202 _Accepted_ - Success.
+- 400 _Bad Request_ - Unknown migration for `MIGRATION_ID` OR Unknown migration step `MIGRATION_STEP_ID` for `MIGRATION_ID`.
```

Comment:
Same remark, isn't it a 404?
And keep the same description as fr the others? "Migration with the given `MIGRATION_ID` or migration step with the given `MIGRATION_STEP_ID` does not exist for `MIGRATION_ID`."

Reply:
Same here with the 2 variations if the `MIGRATION_ID` or the `MIGRATION_STEP_ID` is wrong.

Reply:
Migration with the given `MIGRATION_ID` or `MIGRATION_STEP_ID` does not exist.